internal/static/i18n/de.yaml

```diff
@@ -59,6 +59,19 @@ Errors:
         NotExisting: Multifaktor OTP (OneTimePassword) existiert nicht
         NotReady: Multifaktor OTP (OneTimePassword) ist nicht bereit
         InvalidCode: Code ist ungültig
+      U2F:
+        NotExisting: U2F existiert nicht
```

Comment:
should we describe u2f

Reply:
IMHO an error message should be this short